### ISSUE TITLE
Theme is broken in Emacs 24.4.1

### ISSUE THREAD

**Issue by bradleywright** [sellout/emacs-color-theme-solarized#135]
When I do:

`(load-theme 'solarized)`

in my Emacs 24.4.1, I get the following traceback:

```
Debugger entered--Lisp error: (void-function copy-list)
  copy-list((:foreground base0 :background back))
  solarized-face-for-index((:foreground base0 :background back) 1)
  create-face-spec(default (:foreground base0 :background back))
  apply(create-face-spec (default (:foreground base0 :background back)))
  #[(face) "\301\302^H\"\207" [face apply create-face-spec] 3]((default (:foreground base0 :background back)))
  mapcar(#[(face) "\301\302^H\"\207" [face apply create-face-spec] 3] ((default (:foreground base0 :background back)) (cursor (:foreground base03 :ba$
  solarized-color-definitions()
  (apply (quote custom-theme-set-faces) (quote solarized) (solarized-color-definitions))
  (progn (custom-declare-theme (quote solarized) (quote solarized-theme) solarized-description) (apply (quote custom-theme-set-faces) (quote solarize$
  eval-buffer()  ; Reading at buffer position 250
  load-theme(solarized nil nil)
  #<subr call-interactively>(load-theme record nil)
  funcall(#<subr call-interactively> load-theme record nil)
  (with-no-warnings (funcall ad--addoit-function function record-flag keys))
  (setq ad-return-value (with-no-warnings (funcall ad--addoit-function function record-flag keys)))
  (let ((ido-ubiquitous-next-override (ido-ubiquitous-get-command-override function))) (setq ad-return-value (with-no-warnings (funcall ad--addoit-fu$
  (ido-ubiquitous-with-override (ido-ubiquitous-get-command-override function) (setq ad-return-value (with-no-warnings (funcall ad--addoit-function f$
  (let (ad-return-value) (ido-ubiquitous-with-override (ido-ubiquitous-get-command-override function) (setq ad-return-value (with-no-warnings (funcal$
  ad-Advice-call-interactively(#<subr call-interactively> load-theme record nil)
  apply(ad-Advice-call-interactively #<subr call-interactively> (load-theme record nil))
  call-interactively(load-theme record nil)
  command-execute(load-theme record)
  execute-extended-command(nil "load-theme")
  smex-read-and-run(("toggle-debug-on-error" "emacs-version" "load-theme" "popwin:close-popup-window" "er/expand-region" "package-install" "imenu" "t$
  smex()
  #<subr call-interactively>(smex nil nil)
  funcall(#<subr call-interactively> smex nil nil)
  (with-no-warnings (funcall ad--addoit-function function record-flag keys))
  (setq ad-return-value (with-no-warnings (funcall ad--addoit-function function record-flag keys)))
  (let ((ido-ubiquitous-next-override (ido-ubiquitous-get-command-override function))) (setq ad-return-value (with-no-warnings (funcall ad--addoit-fu$
  (ido-ubiquitous-with-override (ido-ubiquitous-get-command-override function) (setq ad-return-value (with-no-warnings (funcall ad--addoit-function f$
  (let (ad-return-value) (ido-ubiquitous-with-override (ido-ubiquitous-get-command-override function) (setq ad-return-value (with-no-warnings (funcal$
  ad-Advice-call-interactively(#<subr call-interactively> smex nil nil)
  apply(ad-Advice-call-interactively #<subr call-interactively> (smex nil nil))
  call-interactively(smex nil nil)
  command-execute(smex)
```

`copy-list` has been renamed `cl-copy-list` in Emacs 24.4 I think.

**Comment by sellout** [comment]
Whoops – got a little overzealous last night. Checked for the `cl-` prefix, but that wasn't good enough.

I'll revert as soon as I'm at my computer.

**Comment by sellout** [comment]
Fixed in 602cf3e02868c1f782d7f9d504737603f1478bd5.
